packages/adblocker/src/engine/engine.ts

```diff
@@ -310,14 +329,18 @@ export default class FilterEngine extends EventEmitter<
     // Optionally initialize the engine with filters
     cosmeticFilters = [],
     networkFilters = [],
+    preprocessors = [],
```

Comment:
Nit. Not sure if we want to extend this API to allow specifying preprocessors directly; I think it might make more sense to have them as an internal implementation detail for now.

Reply:
Comfirmed. This part is now removed. :+1: